test/s3.test.js

```diff
@@ -772,3 +772,11 @@ tape('accepts region in tilejson', function(assert) {
         AWS.S3 = S3client;
     });
 });
+
+tape('provides requested region partitions', function(assert) {
```

Comment:
Instead of a test that the regionParitions function works, we should be testing that a URI passed in under the china condition comes out with the right host.